### ISSUE TITLE
Cannot start dirigible in tomcat8

### ISSUE THREAD

**Issue by colameo** [eclipse/dirigible#336]
Environment:
- apache-tomcat-8.5.34
- java version "1.8.0_162"
  Java(TM) SE Runtime Environment (build 1.8.0_162-b12)
  Java HotSpot(TM) 64-Bit Server VM (build 25.162-b12, mixed mode)
- mac osx 10.12.6

When I deploy the ROOT.war in tomcat and start catalina I get the following exception:

```
2018-09-22 22:22:48.604 [INFO ] [http-nio-8080-exec-5] o.a.c.i.LoggingInInterceptor - Inbound Message

ID: 19
Address: http://localhost:8080/services/v3/js/ide/services/editors.js
Http-Method: GET
Content-Type:
Headers: {Accept=[*/*], accept-encoding=[gzip, deflate, br], accept-language=[en-GB,en-US;q=0.9,en;q=0.8], connection=[keep-alive], Content-Type=[null], cookie=[JSESSIONID=2A4ED4B789A8ED4F228623B0AA2F12AB], host=[localhost:8080], referer=[http://localhost:8080/services/v3/web/ide/], user-agent=[Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Safari/537.36]}
--------------------------------------
2018-09-22 22:22:49.035 [ERROR] [http-nio-8080-exec-5] o.e.dirigible.api.v3.core.JavaFacade - null
java.lang.reflect.InvocationTargetException: null
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_162]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_162]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_162]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_162]
	at org.eclipse.dirigible.api.v3.core.JavaFacade.call(JavaFacade.java:97) ~[dirigible-api-facade-core-3.2.8-SNAPSHOT.jar:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_162]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_162]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_162]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_162]
	at org.mozilla.javascript.MemberBox.invoke(MemberBox.java:126) [rhino-1.7.7.1.jar:1.7.7.1]
	at org.mozilla.javascript.NativeJavaMethod.call(NativeJavaMethod.java:225) [rhino-1.7.7.1.jar:1.7.7.1]
	at org.mozilla.javascript.optimizer.OptRuntime.callN(OptRuntime.java:52) [rhino-1.7.7.1.jar:1.7.7.1]
	at org.mozilla.javascript.gen.core_v3_java_js_3._c_anonymous_1(core/v3/java.js:15) [rhino-1.7.7.1.jar:na]
	at org.mozilla.javascript.gen.core_v3_java_js_3.call(core/v3/java.js) [rhino-1.7.7.1.jar:na]
	at org.mozilla.javascript.optimizer.OptRuntime.callN(OptRuntime.java:52) [rhino-1.7.7.1.jar:1.7.7.1]
	at org.mozilla.javascript.gen.core_v3_extensions_js_2._c_anonymous_1(core/v3/extensions.js:14) [rhino-1.7.7.1.jar:na]
	at org.mozilla.javascript.gen.core_v3_extensions_js_2.call(core/v3/extensions.js) [rhino-1.7.7.1.jar:na]
	at org.mozilla.javascript.optimizer.OptRuntime.call1(OptRuntime.java:32) [rhino-1.7.7.1.jar:1.7.7.1]
	at org.mozilla.javascript.gen.ide_services_editors_js_1._c_script_0(ide/services/editors.js:14) [rhino-1.7.7.1.jar:na]
	at org.mozilla.javascript.gen.ide_services_editors_js_1.call(ide/services/editors.js) [rhino-1.7.7.1.jar:na]
	at org.mozilla.javascript.ContextFactory.doTopCall(ContextFactory.java:393) [rhino-1.7.7.1.jar:1.7.7.1]
	at org.mozilla.javascript.ScriptRuntime.doTopCall(ScriptRuntime.java:3282) [rhino-1.7.7.1.jar:1.7.7.1]
	at org.mozilla.javascript.gen.ide_services_editors_js_1.call(ide/services/editors.js) [rhino-1.7.7.1.jar:na]
	at org.mozilla.javascript.gen.ide_services_editors_js_1.exec(ide/services/editors.js) [rhino-1.7.7.1.jar:na]
	at org.mozilla.javascript.Context.evaluateReader(Context.java:1250) [rhino-1.7.7.1.jar:1.7.7.1]
	at org.eclipse.dirigible.engine.js.rhino.processor.RhinoJavascriptEngineExecutor.executeService(RhinoJavascriptEngineExecutor.java:164) [dirigible-engine-javascript-rhino-3.2.8-SNAPSHOT.jar:na]
	at org.eclipse.dirigible.engine.js.rhino.processor.RhinoJavascriptEngineExecutor.executeServiceModule(RhinoJavascriptEngineExecutor.java:87) [dirigible-engine-javascript-rhino-3.2.8-SNAPSHOT.jar:na]
	at org.eclipse.dirigible.engine.js.processor.JavascriptEngineProcessor.executeService(JavascriptEngineProcessor.java:42) [dirigible-engine-javascript-3.2.8-SNAPSHOT.jar:na]
	at org.eclipse.dirigible.engine.js.service.JavascriptEngineRestService.executeService(JavascriptEngineRestService.java:71) [dirigible-engine-javascript-3.2.8-SNAPSHOT.jar:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_162]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_162]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_162]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_162]
	at org.apache.cxf.service.invoker.AbstractInvoker.performInvocation(AbstractInvoker.java:180) [cxf-core-3.1.11.jar:3.1.11]
	at org.apache.cxf.service.invoker.AbstractInvoker.invoke(AbstractInvoker.java:96) [cxf-core-3.1.11.jar:3.1.11]
	at org.apache.cxf.jaxrs.JAXRSInvoker.invoke(JAXRSInvoker.java:189) [cxf-rt-frontend-jaxrs-3.1.11.jar:3.1.11]
	at org.apache.cxf.jaxrs.JAXRSInvoker.invoke(JAXRSInvoker.java:99) [cxf-rt-frontend-jaxrs-3.1.11.jar:3.1.11]
	at org.apache.cxf.interceptor.ServiceInvokerInterceptor$1.run(ServiceInvokerInterceptor.java:59) [cxf-core-3.1.11.jar:3.1.11]
	at org.apache.cxf.interceptor.ServiceInvokerInterceptor.handleMessage(ServiceInvokerInterceptor.java:96) [cxf-core-3.1.11.jar:3.1.11]
	at org.apache.cxf.phase.PhaseInterceptorChain.doIntercept(PhaseInterceptorChain.java:308) [cxf-core-3.1.11.jar:3.1.11]
	at org.apache.cxf.transport.ChainInitiationObserver.onMessage(ChainInitiationObserver.java:121) [cxf-core-3.1.11.jar:3.1.11]
	at org.apache.cxf.transport.http.AbstractHTTPDestination.invoke(AbstractHTTPDestination.java:262) [cxf-rt-transports-http-3.1.11.jar:3.1.11]
	at org.apache.cxf.transport.servlet.ServletController.invokeDestination(ServletController.java:234) [cxf-rt-transports-http-3.1.11.jar:3.1.11]
	at org.apache.cxf.transport.servlet.ServletController.invoke(ServletController.java:208) [cxf-rt-transports-http-3.1.11.jar:3.1.11]
	at org.apache.cxf.transport.servlet.ServletController.invoke(ServletController.java:160) [cxf-rt-transports-http-3.1.11.jar:3.1.11]
	at org.apache.cxf.transport.servlet.CXFNonSpringServlet.invoke(CXFNonSpringServlet.java:180) [cxf-rt-transports-http-3.1.11.jar:3.1.11]
	at org.apache.cxf.transport.servlet.AbstractHTTPServlet.handleRequest(AbstractHTTPServlet.java:299) [cxf-rt-transports-http-3.1.11.jar:3.1.11]
	at org.apache.cxf.transport.servlet.AbstractHTTPServlet.doGet(AbstractHTTPServlet.java:223) [cxf-rt-transports-http-3.1.11.jar:3.1.11]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:635) [servlet-api.jar:na]
	at org.apache.cxf.transport.servlet.AbstractHTTPServlet.service(AbstractHTTPServlet.java:274) [cxf-rt-transports-http-3.1.11.jar:3.1.11]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) [catalina.jar:8.5.34]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.34]
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52) [tomcat-websocket.jar:8.5.34]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.34]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.34]
	at org.eclipse.dirigible.runtime.core.filter.HttpContextFilter.doFilter(HttpContextFilter.java:56) [dirigible-service-core-3.2.8-SNAPSHOT.jar:na]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.34]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.34]
	at com.google.inject.servlet.DefaultFilterPipeline.dispatch(DefaultFilterPipeline.java:43) [guice-servlet-3.0.jar:na]
	at com.google.inject.servlet.GuiceFilter.doFilter(GuiceFilter.java:113) [guice-servlet-3.0.jar:na]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.34]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.34]
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:198) [catalina.jar:8.5.34]
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96) [catalina.jar:8.5.34]
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:493) [catalina.jar:8.5.34]
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:140) [catalina.jar:8.5.34]
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81) [catalina.jar:8.5.34]
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:650) [catalina.jar:8.5.34]
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:87) [catalina.jar:8.5.34]
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:342) [catalina.jar:8.5.34]
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:800) [tomcat-coyote.jar:8.5.34]
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66) [tomcat-coyote.jar:8.5.34]
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:806) [tomcat-coyote.jar:8.5.34]
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1498) [tomcat-coyote.jar:8.5.34]
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) [tomcat-coyote.jar:8.5.34]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [na:1.8.0_162]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [na:1.8.0_162]
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) [tomcat-util.jar:8.5.34]
	at java.lang.Thread.run(Thread.java:748) [na:1.8.0_162]
Caused by: org.eclipse.dirigible.core.extensions.api.ExtensionsException: There is no an ExtensionPoint with name [ide-editor] at all.
	at org.eclipse.dirigible.core.extensions.service.ExtensionsCoreService.getExtensionsByExtensionPoint(ExtensionsCoreService.java:345) ~[dirigible-core-extensions-3.2.8-SNAPSHOT.jar:na]
	at org.eclipse.dirigible.api.v3.core.ExtensionsServiceFacade.getExtensions(ExtensionsServiceFacade.java:45) ~[dirigible-api-facade-core-3.2.8-SNAPSHOT.jar:na]
	... 79 common frames omitted
2018-09-22 22:22:49.051 [ERROR] [http-nio-8080-exec-5] o.e.d.e.j.r.p.RhinoJavascriptEngineExecutor - TypeError: Cannot read property "length" from null (ide/services/editors.js#15)
2018-09-22 22:22:49.057 [ERROR] [http-nio-8080-exec-5] o.e.d.e.j.s.JavascriptEngineRestService - org.mozilla.javascript.EcmaError: TypeError: Cannot read property "length" from null (ide/services/editors.js#15)
org.eclipse.dirigible.commons.api.scripting.ScriptingException: org.mozilla.javascript.EcmaError: TypeError: Cannot read property "length" from null (ide/services/editors.js#15)
	at org.eclipse.dirigible.engine.js.rhino.processor.RhinoJavascriptEngineExecutor.executeService(RhinoJavascriptEngineExecutor.java:177) ~[dirigible-engine-javascript-rhino-3.2.8-SNAPSHOT.jar:na]
	at org.eclipse.dirigible.engine.js.rhino.processor.RhinoJavascriptEngineExecutor.executeServiceModule(RhinoJavascriptEngineExecutor.java:87) ~[dirigible-engine-javascript-rhino-3.2.8-SNAPSHOT.jar:na]
	at org.eclipse.dirigible.engine.js.processor.JavascriptEngineProcessor.executeService(JavascriptEngineProcessor.java:42) ~[dirigible-engine-javascript-3.2.8-SNAPSHOT.jar:na]
	at org.eclipse.dirigible.engine.js.service.JavascriptEngineRestService.executeService(JavascriptEngineRestService.java:71) ~[dirigible-engine-javascript-3.2.8-SNAPSHOT.jar:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_162]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_162]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_162]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_162]
	at org.apache.cxf.service.invoker.AbstractInvoker.performInvocation(AbstractInvoker.java:180) [cxf-core-3.1.11.jar:3.1.11]
	at org.apache.cxf.service.invoker.AbstractInvoker.invoke(AbstractInvoker.java:96) [cxf-core-3.1.11.jar:3.1.11]
	at org.apache.cxf.jaxrs.JAXRSInvoker.invoke(JAXRSInvoker.java:189) [cxf-rt-frontend-jaxrs-3.1.11.jar:3.1.11]
	at org.apache.cxf.jaxrs.JAXRSInvoker.invoke(JAXRSInvoker.java:99) [cxf-rt-frontend-jaxrs-3.1.11.jar:3.1.11]
	at org.apache.cxf.interceptor.ServiceInvokerInterceptor$1.run(ServiceInvokerInterceptor.java:59) [cxf-core-3.1.11.jar:3.1.11]
	at org.apache.cxf.interceptor.ServiceInvokerInterceptor.handleMessage(ServiceInvokerInterceptor.java:96) [cxf-core-3.1.11.jar:3.1.11]
	at org.apache.cxf.phase.PhaseInterceptorChain.doIntercept(PhaseInterceptorChain.java:308) [cxf-core-3.1.11.jar:3.1.11]
	at org.apache.cxf.transport.ChainInitiationObserver.onMessage(ChainInitiationObserver.java:121) [cxf-core-3.1.11.jar:3.1.11]
	at org.apache.cxf.transport.http.AbstractHTTPDestination.invoke(AbstractHTTPDestination.java:262) [cxf-rt-transports-http-3.1.11.jar:3.1.11]
	at org.apache.cxf.transport.servlet.ServletController.invokeDestination(ServletController.java:234) [cxf-rt-transports-http-3.1.11.jar:3.1.11]
	at org.apache.cxf.transport.servlet.ServletController.invoke(ServletController.java:208) [cxf-rt-transports-http-3.1.11.jar:3.1.11]
	at org.apache.cxf.transport.servlet.ServletController.invoke(ServletController.java:160) [cxf-rt-transports-http-3.1.11.jar:3.1.11]
	at org.apache.cxf.transport.servlet.CXFNonSpringServlet.invoke(CXFNonSpringServlet.java:180) [cxf-rt-transports-http-3.1.11.jar:3.1.11]
	at org.apache.cxf.transport.servlet.AbstractHTTPServlet.handleRequest(AbstractHTTPServlet.java:299) [cxf-rt-transports-http-3.1.11.jar:3.1.11]
	at org.apache.cxf.transport.servlet.AbstractHTTPServlet.doGet(AbstractHTTPServlet.java:223) [cxf-rt-transports-http-3.1.11.jar:3.1.11]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:635) [servlet-api.jar:na]
	at org.apache.cxf.transport.servlet.AbstractHTTPServlet.service(AbstractHTTPServlet.java:274) [cxf-rt-transports-http-3.1.11.jar:3.1.11]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) [catalina.jar:8.5.34]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.34]
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52) [tomcat-websocket.jar:8.5.34]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.34]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.34]
	at org.eclipse.dirigible.runtime.core.filter.HttpContextFilter.doFilter(HttpContextFilter.java:56) [dirigible-service-core-3.2.8-SNAPSHOT.jar:na]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.34]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.34]
	at com.google.inject.servlet.DefaultFilterPipeline.dispatch(DefaultFilterPipeline.java:43) [guice-servlet-3.0.jar:na]
	at com.google.inject.servlet.GuiceFilter.doFilter(GuiceFilter.java:113) [guice-servlet-3.0.jar:na]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.34]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.34]
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:198) [catalina.jar:8.5.34]
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96) [catalina.jar:8.5.34]
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:493) [catalina.jar:8.5.34]
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:140) [catalina.jar:8.5.34]
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81) [catalina.jar:8.5.34]
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:650) [catalina.jar:8.5.34]
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:87) [catalina.jar:8.5.34]
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:342) [catalina.jar:8.5.34]
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:800) [tomcat-coyote.jar:8.5.34]
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66) [tomcat-coyote.jar:8.5.34]
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:806) [tomcat-coyote.jar:8.5.34]
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1498) [tomcat-coyote.jar:8.5.34]
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) [tomcat-coyote.jar:8.5.34]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [na:1.8.0_162]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [na:1.8.0_162]
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) [tomcat-util.jar:8.5.34]
	at java.lang.Thread.run(Thread.java:748) [na:1.8.0_162]
Caused by: org.mozilla.javascript.EcmaError: TypeError: Cannot read property "length" from null (ide/services/editors.js#15)
	at org.mozilla.javascript.ScriptRuntime.constructError(ScriptRuntime.java:3951) ~[rhino-1.7.7.1.jar:1.7.7.1]
	at org.mozilla.javascript.ScriptRuntime.constructError(ScriptRuntime.java:3929) ~[rhino-1.7.7.1.jar:1.7.7.1]
	at org.mozilla.javascript.ScriptRuntime.typeError(ScriptRuntime.java:3962) ~[rhino-1.7.7.1.jar:1.7.7.1]
	at org.mozilla.javascript.ScriptRuntime.typeError2(ScriptRuntime.java:3981) ~[rhino-1.7.7.1.jar:1.7.7.1]
	at org.mozilla.javascript.ScriptRuntime.undefReadError(ScriptRuntime.java:3993) ~[rhino-1.7.7.1.jar:1.7.7.1]
	at org.mozilla.javascript.ScriptRuntime.getObjectProp(ScriptRuntime.java:1505) ~[rhino-1.7.7.1.jar:1.7.7.1]
	at org.mozilla.javascript.gen.ide_services_editors_js_1._c_script_0(ide/services/editors.js:15) ~[na:na]
	at org.mozilla.javascript.gen.ide_services_editors_js_1.call(ide/services/editors.js) ~[na:na]
	at org.mozilla.javascript.ContextFactory.doTopCall(ContextFactory.java:393) ~[rhino-1.7.7.1.jar:1.7.7.1]
	at org.mozilla.javascript.ScriptRuntime.doTopCall(ScriptRuntime.java:3282) ~[rhino-1.7.7.1.jar:1.7.7.1]
	at org.mozilla.javascript.gen.ide_services_editors_js_1.call(ide/services/editors.js) ~[na:na]
	at org.mozilla.javascript.gen.ide_services_editors_js_1.exec(ide/services/editors.js) ~[na:na]
	at org.mozilla.javascript.Context.evaluateReader(Context.java:1250) ~[rhino-1.7.7.1.jar:1.7.7.1]
	at org.eclipse.dirigible.engine.js.rhino.processor.RhinoJavascriptEngineExecutor.executeService(RhinoJavascriptEngineExecutor.java:164) ~[dirigible-engine-javascript-rhino-3.2.8-SNAPSHOT.jar:na]
	... 53 common frames omitted
2018-09-22 22:22:49.082 [INFO ] [http-nio-8080-exec-5] o.a.c.i.LoggingOutInterceptor - Outbound Message
---------------------------
ID: 19
Response-Code: 500
Content-Type: application/octet-stream
Headers: {Content-Type=[application/octet-stream], Date=[Sat, 22 Sep 2018 20:22:49 GMT]}
Messages: --- Binary Content ---
```
`

**Comment by delchev** [comment]
Hi Serano,

In fact, you have to wait a few seconds (15-20) for Dirigible to get fully initialised. There are several synchroniser jobs that have to pass at least ones to register all the needed pre-delivered content.
Please, let me know if after a while you can access the WebIDE and everything is fully working.

We will consider implementing more descriptive status message in this case.

Regards,
Nedelcho

**Comment by colameo** [comment]
Hi Nedelcho,
thanks, works!
Best
Serano

**Comment by ThuF** [comment]
Hi Serano,

We've provided a fix for the startup exceptions (ff17638827acab1d61b496411b3ca00faf5bead2), can you confirm that with this fix we can close the issue?

Regards,
Yordan

**Comment by colameo** [comment]
Hello Yordan,
I've merged your changes and it works now.
Thanks
Serano
